src/components/FooterComponent/index.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+import Footer from './components/Footer'
+
+
+
+export default Footer
```

Comment:
```suggestion
import React from 'react'
import Footer from './components/Footer'


```
export default Footer
```